src/App.scss

```diff
@@ -164,28 +164,47 @@ body {
     grid-column-end: 3;
     grid-row-start: 2;
     grid-row-end: 4;
-    display: grid;
+    display: flex;
+    flex-direction: column;
     grid-template-rows: min-content min-content auto;
 }
 
 .App__command-palette-command-container {
     background-color: #30444E;
     border-radius: 3px;
-    width: 100%;
     height: 100%;
+    width: 100%;
     overflow-y: auto;
     grid-row-start: 3;
     grid-row-end: 4;
     padding-top: 1rem;
 }
 
-.App__command-palette-commands {
-    width: 6rem;
+.App__command-palette-section-heading {
+    border-bottom: 1px solid #818A98;
```

Comment:
This border is only applied underneath the text, I think it would be cleaner if we wrap the headings in a container and apply border-bottom on the container.

Reply:
We can have a container for the headings("Movements" and "Controls") that has margin top and bottom to make them more centered. Right now they are not vertically centered; Movements have more space at the top and Controls have more space at the bottom. After adding a container for them, we need to remove some padding from existing containers.

Reply:
We can give some margin-left in the container I suggested to push the text little bit towards right.